presto-common/src/main/java/com/facebook/presto/common/block/ClosingBlockLease.java

```diff
@@ -13,16 +13,13 @@
  */
 package com.facebook.presto.common.block;
 
-import javax.annotation.concurrent.NotThreadSafe;
-
 import java.util.List;
 
 import static java.util.Arrays.asList;
 import static java.util.Collections.emptyList;
 import static java.util.Collections.unmodifiableList;
 import static java.util.Objects.requireNonNull;
 
-@NotThreadSafe
```

Comment:
Shouldn't we keep this annotation?

Reply:
We actually have a replacement of `@NotThreadSafe` inside of `com.facebook.airlift:concurrent` now, however, `presto-common` doesn't have it as a dependency. I didn't want to add a new dependency in `presto-common` just to add an annotation for documentation purposes, so I opted to just remove the annotation instead. We could consider just adding a javadoc mentioning that it is not thread safe as a replacement?